spec/controllers/password_retrieval_controller_spec.rb

```diff
@@ -40,10 +40,10 @@
   describe 'check if token is expired' do
     it 'checks when token is expired' do
       local_token = 'some random string'
-      @password_retrival = PasswordReset.new
-      @password_retrival.token = Digest::SHA1.hexdigest(local_token)
-      @password_retrival.user_email = 'example@example.edu'
-      @password_retrival.save!
+      @password_retreival = PasswordReset.new
```

Comment:
There's a typo in `password_retreival`. The correct spelling should be `password_retrieval`.